### ISSUE TITLE
"Donate for afloat" opens repeatedly

### ISSUE THREAD

**Issue by madvas** [millenomi/afloat#19]
After every boot, when I open application e.g Preview for the first time, it always opens this dialog:

![screen shot 2015-05-09 at 10 45 23](https://cloud.githubusercontent.com/assets/3857155/7549518/86f6db5a-f638-11e4-9f10-549ae426c3ca.png)

I click open and nothing happens. When I restart machine, it happens again. It's quite annoying. Happens with other applications too, but not with every. E.g with TextEdit it does it also. I'm using Yosemite. and Afloat 2.1.1

**Comment by gmogames** [comment]
Yep, happening here as well. Would love to know how to stop this from happening. Clicking on any option doesn't do anything and I even have afloat uninstalled and it keeps showing up.

**Comment by mrfrankus** [comment]
Hi there, 

I am curious… if you click on “Show Application” where does it take you?  Afloat, or perhaps SIMBL (or rather, whatever middleman you’re relaying on)?  I myself have the same Afloat I compiled ages ago (way before 10.9, possibly even earlier) and I have no such a popup.

f.
On May 14, 2015 at 13:28:13, AppWish (notifications@github.com) wrote:

Yep, happening here as well. Would love to know how to stop this from happening. Clicking on any option doesn't do anything and I even have afloat uninstalled and it keeps showing up.

—
Reply to this email directly or view it on GitHub.

 
👤 Frank • gravatar.com/mrfrankus

**Comment by madvas** [comment]
When I click "Show Application" this is what it opens

![screen shot 2015-05-14 at 22 18 53](https://cloud.githubusercontent.com/assets/3857155/7641029/6d15f84e-fa87-11e4-8d8f-c61e92ce6984.png)

**Comment by mrfrankus** [comment]
If I look into my Xcode project folder, I do see Donate files in there, however none of these ended up in the final/installed bundle, which looks rather very different from yours indeed: 

This is what my Info.plist looks like: 

Could you send me a copy of your Info.plist from your installed bundle?  
(You might need to rename the extension else it could get blocked)

I get a feeling there will be hooks in yours that enable the loading of the donation alert.
Note: I don’t want to be perceived as aiding in the disabling of donation nagware if this was intended to be part of the final product, so please don’t hesitate to chime in if I should refrain from pursuing this line of inquiry.

f.

On May 14, 2015 at 16:21:07, Matus Lestan (notifications@github.com) wrote:

When I click "Show Application" this is what it opens

—
Reply to this email directly or view it on GitHub.

 
👤 Frank • gravatar.com/mrfrankus

**Comment by gmogames** [comment]
My case, I completely deleted the afloat bundle, all afloat files I found in the hard drive, did a search for any "afloat" mentions inside of files, but I still keep getting this annoying donate for afloat window.

**Comment by aangelinsf** [comment]
Same problem here...but happens not just when I reboot. Randomly happens during normal work.

**Comment by tjementum** [comment]
Me to.. quite annoying. I would even donate to fix this... but nothing happens when I start the Donate app.

BTW: I installed Afloat [using EasySIMBL buy following this guide](http://www.perfectlyrandom.org/2015/01/31/always-on-top-in-macos-yosemite/) and a comment here also showed a solution (I hope/think**). 
1. Start **EasySIMBL**.
2. Click **Show Plugin Folder** and navigate to the **Contents** folder.
3. Right click **Afloat.bundle** and click **Show Package Content**.
4. Delete the **Donate for Afloat** app.
5. Navigate into the **Resoure/it.lproj** folder.
6. Delete **Donate.nib**, **AfloatDonateFirstAlert.strings** and **AfloatDonateLastAlert.strings**.

The content of the it.lproj and en.lproj folder now contains the same files (different sizes).

*\* I have not see the popup since. But I'm also not sure exactly when I got this popup. If it is not fixed of if I get other issues I'll promise to write here again. If I don't I consider this the solution :)

**Comment by gmogames** [comment]
I was able to fix the issue, and it's kind of the same solution as @tjementum .

Even after deleting everything related to afloat, I still had EasySIMBL installed. I opened and found a tab in there where there was another instance of afloat.
I deleted and voila, no more annoying popup!

**Comment by madvas** [comment]
@tjementum and after this solution "Keep Afloat" functionality still works?

**Comment by tjementum** [comment]
@madvas Yes... it still works. And I have not seen a popup in since.

**Comment by kachizukanne** [comment]
Thanks @tjementum, works. Also worked for these guys [here](http://www.perfectlyrandom.org/2015/01/31/always-on-top-in-macos-yosemite/)
